blueocean-i18n/src/main/java/io/jenkins/blueocean/i18n/BlueI18n.java

```diff
@@ -136,12 +136,19 @@ public HttpResponse doDynamic(StaplerRequest request) {
     private JSONObject getBundle(BundleParams bundleParams, Locale locale) {
         PluginWrapper plugin = bundleParams.getPlugin();
```

Comment:
so is this because when using a sharable module the plugin is the name in the jar and hard coded at build time (so we serve js files that reference the jar's artifact ID rather than the plugins artifact IDs instead of the plugin that includes the jar?)